docs/source/jobs.rst

```diff
@@ -299,25 +401,12 @@ For example, to open the store in read-only mode, you would write:
     ...     print(job.data.x)
 
 Explicitly opening the underlying file by either using the context manager or the ``open()`` function is required when reading and writing arrays, such as ``numpy.arrays``.
-Please see the next section for details on accessing arrays.
+Please see the **Accessing arrays** section for details on accessing arrays.
 
 .. warning::
 
     It is strongly advised that operations on :py:attr:`Job.data` are not performed in parallel, to avoid data corruption.
```

Comment:
```suggestion
    It is recommended to use [file locks](https://pypi.org/project/filelock/) when accessing HDF5-files in parallel, *i.e.*, from multiple operations that are independent from each other.
```
I recently learned about the recommendation for *affirmative tone* as part of the [Python documentation guide](https://devguide.python.org/documenting/#affirmative-tone). I suggest that we slightly adjust this warning accordingly.

As a side note, eventually we should probably implement a signac-inherit locking mechanism.